packages/chat-component-bindings/src/messageThreadSelector.ts

```diff
@@ -124,9 +128,10 @@ export const messageThreadSelector: MessageThreadSelector = createSelector(
             message.type.toLowerCase() === ACSKnownMessageType.text ||
             message.type.toLowerCase() === ACSKnownMessageType.richtextHtml ||
             message.type.toLowerCase() === ACSKnownMessageType.html ||
-            message.type === ACSKnownMessageType.participantAdded ||
-            message.type === ACSKnownMessageType.participantRemoved ||
-            message.type === ACSKnownMessageType.topicUpdated ||
+            (message.type === ACSKnownMessageType.participantAdded && hasValidParticipant(message)) ||
+            (message.type === ACSKnownMessageType.participantRemoved && hasValidParticipant(message)) ||
+            // TODO: Add support for topicUpdated system messages in MessageThread component.
+            // message.type === ACSKnownMessageType.topicUpdated ||
```

Comment:
Uncomment until we fix this TODO?

Reply:
We don't want it until the todo is fixed (otherwise displays empty messages), happy to delete the line instead